src/process/message_processor.go

```
@@ -0,0 +1,44 @@
+package process
+
+import (
+	"BGPAlert/common"
+	"fmt"
+	"time"
+)
+
+// Constantly reads channel of messages and stores them in Window objects to send through windowChannel for analysis
+func ProcessBGPMessages(msgChannel chan common.BGPMessage, windowChannel chan common.Window) {
+	var bucketMap = make(map[time.Time][]common.BGPMessage)
+
+	for msg := range msgChannel {
+
+		windowSize, _ := time.ParseDuration("60s")
```

If it safe to ignore error here?